DataEngineering/setup.py

```diff
@@ -0,0 +1,10 @@
+import mysql.connector as mysql
+db = mysql.connect(user='root', password='',host='localhost')
+cursor = db.cursor()
+cursor.execute("create database db1;")
+cursor.execute("use db1;")
+createDB = "create table data(  id varchar(255),  commonName varchar(255), lat float, lon float, time TIME, NbBikes varchar(20),primary key(id,time));"
```

Comment:
Though it wasn't discussed as a part of the requirements, it might also be a good idea to include NbDocks here as well. As this might change over time, a good idea would be to take the ratio of Bikes available to total docks over time as a signal for the ML model to predict available bikes.